build/common.targets

```diff
@@ -196,6 +196,36 @@
 
   <!--
     ============================================================
+    GetLocalizedFiles - gathers the list of files that have been
+    localized from each project
+    ============================================================
+  -->
+  <!--Target is invoked by build\loc.proj\MoveLocalizedFilesToLocalizedArtifacts to get all the localized files from all projects-->
+  <Target Name="GetLocalizedFiles" DependsOnTargets="GetTargetFrameworkSet" Returns="@(LocalizedFiles)" Condition=" '$(Shipping)' == 'true' ">
+    <MSBuild
+      Projects="$(MSBuildProjectFullPath)"
+      Targets="GetLocalizedFilesFromProject"
+      Properties="TargetFramework=%(ProjectTargetFrameworkEntries.Identity);
```

Comment:
All the localised files are already present in $(LocalizationRootDirectory) , this move is redundant